Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,338 @@
+                                Console.WriteLine("There are not customers older to 55 yeards old");                
+                    }
+                }
+        }
+
+        public static void CustomerReportAge(List<Customer> customersList)
+        {
+            bool flag = true;
+            DateTime date = DateTime.Today;
+            string monthString = date.ToString("MMMM");
+            Console.WriteLine($" Customers who meet years int the month of : {monthString}");           
+            int month = DateTime.Today.Month;
+            if(customersList.Count == 0)
+            {
+                Console.WriteLine("The are not registered customers.");
+            }
+            else
+                {
+                    foreach (Customer clients in customersList)
+                    {
+                            if (month == clients.MonthBirthday)
+                            {   
+                                flag = false;
+                                Console.WriteLine("--------------");
+                                Console.WriteLine("Customer full name :" + $"{clients.FullName} {clients.LastName}, Age : {clients.Age}, Birthday Month : {clients.MonthBirthday}");                        
+                            }
+                            if (flag)
+                                Console.WriteLine($"Any customer meet years in the month of {monthString}");           
+                    }
+                }
+        }
+
+        public static void ProductsReport(List<Product> productsList)
+        {
+
+            Console.WriteLine("Products with stock is minimum");
+            if(productsList.Count == 0)
+            {
+                Console.WriteLine("There are not registered products.");
+            }
+            else
+                {
+                    foreach (Product Products in productsList)
+                    {              
+                        if (Products.Stock < MINIMO_STOCK)
+                        {
+                            Console.WriteLine("--------------");
+                            Console.WriteLine("Product name : " + $"{Products.NameProduct}, Price : {Products.Cost}, Stock : {Products.Stock}");                  
+                        }
+                        else
+                            Console.WriteLine("The are not products with stock less at all them");
+                    }
+                }
+        }
+
+        public static void CustomerRegistration(List<Customer> customersList)
+        {
+            int number;
+            bool answer;
+            Customer clienteNuevo = new Customer();
+            Console.WriteLine("---------------------");
+            Console.WriteLine("Customer Registration");
+            Console.WriteLine("---------------------");
+            do 
+            {
+                Console.WriteLine("Enter customer DNI :");
+                answer = Int32.TryParse(Console.ReadLine(),out number);

```

Comment:
answer = Int32.TryParse(Console.ReadLine(), out number);
after a comma you have to use an blank space.